pkg/metrics/metrics.jsx

```diff
@@ -1213,14 +1213,17 @@ class MetricsHistory extends React.Component {
 }
 
 export const Application = () => (
-    <Page groupProps={{ sticky: 'top' }}
-          isBreadcrumbGrouped
-          breadcrumb={
-              <Breadcrumb>
-                  <BreadcrumbItem onClick={() => cockpit.jump("/system")} to="#">{_("Overview")}</BreadcrumbItem>
-                  <BreadcrumbItem isActive>{_("Performance Metrics")}</BreadcrumbItem>
-              </Breadcrumb>
-          }>
+    <Page>
+        <PageSection variant={PageSectionVariants.light}>
```

Comment:
I see two drawbacks with this change still:
* breadcrumb is not longer sticky
* we dont use the default paddings, indeed overview and kdump use the same approach, but all other pages not. 

Definitely not blockers.

Reply:
https://paste.centos.org/view/6c0c0993
This change will keep the header sticky.

Reply:
Nice, thank you! Admittedly I didn't even notice the non-stickyness. Applied.